WhateverGreen/kern_igfx.cpp

```diff
@@ -98,6 +98,7 @@ void IGFX::init() {
 			currentGraphics = &kextIntelKBL;
 			currentFramebuffer = &kextIntelKBLFb;
 			forceCompleteModeset.supported = forceCompleteModeset.enable = true;
+			RPSControl.enabled = true;
```

Comment:
These need to be duplicated for `requresFramebufferPatches` and `requiresGraphicsPatches`, and be disabled on Apple hardware and/or depend on GuC firmware loading.

Reply:
It already depends on GuC loading: the `IGGuC`  scheduler is used in that case, so our hooks would never get invoked. On the other hand, the driver may still decide to fallback to `IGScheduler2`, and our hooks would be in place.